authors/apps/profiles/test_follow.py

```diff
@@ -0,0 +1,108 @@
+from django.contrib.auth import get_user_model
+from rest_framework.test import APITestCase
+from rest_framework.reverse import reverse
+from rest_framework import status
+from .models import Profile
+
+
+from authors.apps.authentication.models import UserManager
+
+
+class ProfileTest(APITestCase):
+    """
+    User authentication test cases
+    """
+    def setUp(self):
+        """
+        Method for setting up user
+        """
+        self.user =  { 
+        "user":{
+                'username': 'janeDoe', 
+                'email': 'jane@doe.com', 
+                'password': 'janedoe123', 
+        }
+        }
+
+        self.user_2 =  { 
+        "user":{
+                'username': 'jane1Doe', 
+                'email': 'jane3@doe.com', 
+                'password': 'jane3doe123', 
+        }
+        }
+
+        self.url_register = reverse('authentication:user-registration')
+        reg_res=self.client.post(self.url_register,self.user_2,format='json')
+        # self.url_login = reverse('authentication:user-login')
```

Comment:
Let's lose this comment.

Reply:
thanks @codingedward , I will refrain from this bad habit of having commented code